bool appex
bool terrain
bool two_leg
bool alt_leg
bool left_leg
bool right_leg
bool hip
float64 left_toe
float64 right_toe
bool stand
bool hop
float64 slip_h
float64 stand_r0
float64 slip_r0
float64 slip_m
float64 leg_pos_kp
float64 leg_for_kp
float64 leg_pos_kd
float64 leg_for_kd
float64 hip_kp
float64 hip_kd
float64 robot_ks
float64 robot_kt
float64 robot_kg
